core/src/main/java/org/testcontainers/images/builder/Transferable.java

```diff
@@ -78,4 +89,8 @@ default void transferTo(TarArchiveOutputStream tarArchiveOutputStream, final Str
     default String getDescription() {
         return "";
     }
+
+    default void updateChecksum(Checksum checksum) {
```

Comment:
Can we just remove this, to fix `japicmp` error?

Reply:
What does it say?

Default methods should be fine, and, worst case scenario, we could just allowlist it

Reply:
This is the report:
![image](https://user-images.githubusercontent.com/5088104/162422878-bf7da88c-ecba-4254-a16b-8c4cc33db98f.png)

This is the explanation from Oracle docs on binary compatibility:
> Adding a default method, or changing a method from abstract to default, does not break compatibility with pre-existing binaries, but may cause an IncompatibleClassChangeError if a pre-existing binary attempts to invoke the method.

Reply:
hmm, `of` is clearly false positive (isn't a new default method, but static)

`updateChecksum` can be allow-listed too 👍